scripts/extract_version_from_rpm_spec.py

```diff
@@ -0,0 +1,47 @@
+import re
+
+import click
+
+
+@click.command()
+@click.argument("spec_path")
+def get_convert2rhel_version(spec_path: str) -> None:
+    """Parse rpm spec file and returns its name-version-release part.
+
+    Example:
+    ```bash
+    python scripts/extract_version_from_rpm_spec.py packaging/convert2rhel.spec
+    # convert2rhel-0.21-1
```

Comment:
```suggestion
    # 0.21-1
```
This script name suggests that the output should be version, not name and version.

Technically 0.21-1 is version and release, but that's not that important.